packages/core/src/components/button/_button-group.scss

```diff
@@ -142,17 +142,25 @@ Styleguide button-group
   }
 
   &.#{$ns}-outlined {
-    > .#{$ns}-button {
+    .#{$ns}-button {
```

Comment:
We likely need to keep the child combinator here, otherwise a button inside an inline overlay within the button group would be affected by these styles. We'll need to be specific for popovers, similar to L100-L112 above.